lib/octicons_react/src/index.d.ts

```diff
@@ -1,14 +1,19 @@
+// eslint-disable-next-line import/no-namespace
 import * as React from 'react'
 
-import {Icon} from './__generated__/icons'
+import { Icon } from './__generated__/icons'
 
 type Size = 'small' | 'medium' | 'large'
 
 export interface OcticonProps {
   'aria-label'?: string
+  'aria-labelby'?: string
   tabIndex?: number
   children?: React.ReactElement<any>
   className?: string
+  title?: string | React.ReactElement<any>
+  ref?: React.Ref
```

Comment:
```suggestion
```

I think we might need to exclude `ref` support unless we can find a way to safely use `React.forwardRef` (which seems tricky since this package supports v16)